Sources/Sentry/Profiling/SentryBacktrace.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
Would you be massively opposed to using `.hpp` for C++ headers rather than `.h`? We have been relatively consistent about doing this so far.  Obviously the same with other files.

Reply:
This is for an Objective-C+,+ file would you suggest `.hpp` for those too?
`Sources/Sentry/Profiling/SentryBacktrace.mm`

Reply:
I can change C++ headers to be .hpp, no problem.